mungegithub/mungers/release-note-label.go

```diff
@@ -55,9 +55,10 @@ Please see: https://github.com/kubernetes/community/blob/master/contributors/dev
 )
 
 var (
-	releaseNoteBody       = fmt.Sprintf(releaseNoteFormat, releaseNote, releaseNoteActionRequired, releaseNoteExperimental, releaseNoteNone)
-	parentReleaseNoteBody = fmt.Sprintf(parentReleaseNoteFormat, releaseNote, releaseNoteActionRequired)
-	noteMatcherRE         = regexp.MustCompile(`(?s)(?:Release note\*\*:\s*(?:<!--[^<>]*-->\s*)?` + "```(?:release-note)?|```release-note)(.+?)```")
+	releaseNoteBody           = fmt.Sprintf(releaseNoteFormat, releaseNoteLabelNeeded, releaseNote, releaseNoteActionRequired, releaseNoteExperimental, releaseNoteNone)
+	deprecatedReleaseNoteBody = fmt.Sprintf(releaseNoteFormat, deprecatedReleaseNoteLabelNeeded, releaseNote, releaseNoteActionRequired, releaseNoteExperimental, releaseNoteNone)
```

Comment:
nit: You could simplify these two calls by moving the list of accepted labels into the format string itself.